host/xtest/regression_4000.c

```diff
@@ -5549,6 +5549,11 @@ static void xtest_tee_test_4013(ADBG_Case_t *c)
 ADBG_CASE_DEFINE(regression, 4013, xtest_tee_test_4013,
 		"Test generation of device unique TA keys");
 
+/*
+ * regression_4014
+ * Depends on SM2 KEP support in OP-TEE OS.
```

Comment:
True, and sub-cases in 4006 depend on SM2 PKE or SM2 DSA yet you're not mentioning that... Similarly 4001 will test SM3 if available.

In fact I don't see a reason to add all these comments except for the kernel-related ones to help identify which patches exactly are needed.

Reply:
I agree :) that's why I split into 2 commits.
I propose to discard the 2nd commit and keep only the 1st one. Ok?

Reply:
> I agree :) that's why I split into 2 commits.

Aha true I missed that :) it's good.

> I propose to discard the 2nd commit and keep only the 1st one. Ok?

Sounds good.